### PR TITLE
[5.1] Optional case sensitivity for str_contains helper function

### PR DESCRIPTION
**Pull request by m1so**
Ability to call `str_contains` with 3rd optional parameter for case sensitivity

### CODE REVIEW

File: src/Illuminate/Support/Str.php [hunk 1]
```diff
@@ -62,12 +62,13 @@ public static function camel($value)
      *
      * @param  string  $haystack
      * @param  string|array  $needles
+     * @param  boolean  $caseSensitive  (optional)
```
Comment (GrahamCampbell) [inline src/Illuminate/Support/Str.php]: bool not boolean

Comment (GrahamCampbell) [inline src/Illuminate/Support/Str.php]: also, please drop the optional comment

Comment (m1so) [inline src/Illuminate/Support/Str.php]: my bad, see the new commit

**Comment by phanan** [comment]
See #8581 (rejected).

**Comment by jonnywilliamson** [comment]
Boo!

**Pull request closed without merging**